packages/program-test-suite/src/steps/assertions.ts

```diff
@@ -45,6 +45,26 @@ export function init(cucumber: Cucumber): void {
     });
   });
 
+  Then('the following rewards will exist:', function(this: World, data:any) {
+    const rewards = this.state.programTriggerResult.mutations.filter(
+      (m: any) => {
+        return (
+          m.type === 'CREATE_REWARD'
+        );
+      },
+    );
+
+    data.hashes().forEach((row: MutationStepRow, idx: number) => {
+      const reward = rewards[idx];
+      assert(reward);
+      assert.strictEqual(reward.data.user.id, `${row.user.toUpperCase()}ID`);
```

Comment:
Maybe we should start using assertion error message? https://www.w3schools.com/nodejs/met_assert_strictequal.asp

Seemed like this was a pain during debugging other assertions today.

```suggestion
      assert.strictEqual(reward.data.user.id, `${row.user.toUpperCase()}ID`, "Invalid user");
```